fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/utils/iterators/RdfStream.java

```diff
@@ -359,6 +359,11 @@ public Triple apply(final T prototriple) {
         };
     }
 
+    protected static <From, To> Iterator<To> flatMap(final Iterator<From> i,
```

Comment:
Rather than reimplementing a `flatmap` here (next we'll be implementing `map` and `filter`), why not user `StreamSupport` and the `Iterable` interface to convert the `Iterator` into a `Stream`?